index.js

```diff
@@ -0,0 +1,79 @@
+'use strict';
+var bodyParser = require('body-parser');
+var express = require('express');
+var GitHubServer = require('./lib/GitHubServer');
+var gitHubWebHook = require('express-github-webhook');
+var nconf = require('nconf');
+var Promise = require('bluebird');
+var RegexTools = require('./lib/RegexTools');
+
+var app = express();
+module.exports = app;
+
+nconf.env('__')
```

Comment:
If this is the only file that will ever use `nconf` than this is fine.  But if we need to expose settings in other files, we'll want to change this so that all of the nconf stuff is abstracted away behind a settings object.